### ISSUE TITLE
Import Markdown (Directory) not working

### ISSUE THREAD

**Issue by emilyanncr** [laurent22/joplin#6525]
When I try and use the Import -> Markdown (Directory) option and select the folder of files I'd like to have imported, nothing happens.  Please advise. Thanks.
I've installed Joplin as an application on my Macbook Pro Catalina 10.15.7
Joplin:
Sync Version: 3
Profile Version: 41

## Environment

Joplin version: 
Platform: 
OS specifics: 

## Steps to reproduce

1.  Open the application
2. Click Import -> Markdown (Directory)
3. Select folder of files you'd like to import
4. Click Import

## Describe what you expected to happen

That it would import all the files successfully

**Comment by laurent22** [comment]
How does the directory structure look? It will only import .md files, is that what's in your directory?

Also please include the app version number

**Comment by emilyanncr** [comment]
I exported my notes from Notion in markdown format before trying to import the files into Joplin.
Update:
I was able to complete the import just fine using my Linux machine.  So the issue seems to be tied to my Mac.  As far as the version of Joplin I was using on my Mac, I couldn't find a version number.  All I could find was the sync and Profile Version #s which I included above. It was the most recent one available in the Apple Store.
